cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java

```diff
@@ -141,14 +141,20 @@ private void compile(@CommandLine.Option(names = {"-s", "--source"}) String sour
             FileUtils.deleteDirectory(targetWorkspace.toFile());
             Files.createDirectory(targetWorkspace);
 
-            Translator<Database, Database> translator = TranslatorFactory
-                    .translator(source.getDbType(), target.getDbType());
-
-            translatedModels = getDatabases(sourceWorkspace)
-                    .map(translateDatabases(translator))
-                    .collect(Collectors.toList());
-
-            translatedModels.forEach(writeOutput(targetWorkspace));
+            if (source.getDbType().equals(target.getDbType())) {
+                log.info("Skipping translation because the target ({}) and source ({}) db types are the same.",
```

Comment:
Still checking if we need to add **model.yml** in the target DB, during compile where Source and Target are the same DB Type

Reply:
Added the **model.yml** also